app/src/main/res/menu/activity_main_drawer.xml

```diff
@@ -69,6 +69,10 @@
             android:id="@+id/nav_add_images"
             android:icon="@drawable/ic_add_black_24dp"
             android:title="@string/add_images" />
+        <item>
```

Comment:
this will obviously give an error. Why are you closing the item tag here? This is causing major issue.